layabout.py

```diff
@@ -1,3 +1,287 @@
+import os
+import time
+import logging
+from typing import Any, DefaultDict, Dict, Callable, List, Tuple
+from inspect import signature
+from functools import wraps
+from collections import defaultdict
+
+from slackclient import SlackClient
+
+# TODO: This is a dependency of slackclient needed for exception handling. In
```

Comment:
Oh! Yes, I should be pulling it through `slackclient`. I always forget that's an option! There's an explanation of why next to where I use the import from this library rather than at the import site.

Reply:
It looks like they were really careful not to re-export it anywhere. I can't find an easy way to pull it through `slackclient`... :disappointed:

Reply:
That's kinda dumb. =(